code/Search/SearchForm.php

```diff
@@ -163,7 +163,7 @@ public function getResults()
         $keywords = $this->addStarsToKeywords($keywords);
 
         $pageLength = $this->getPageLength();
-        $start = $request->requestVar('start') ?: 0;
+        $start = max(0, (int)$request->requestVar('start')) ?: 0;
```

Comment:
The ternary operator isn't needed any more

```diff
-$start = max(0, (int)$request->requestVar('start')) ?: 0;
+$start = max(0, (int)$request->requestVar('start'));
```

Reply:
Sure thing. I have made this change